code/obj/machinery/plantpot.dm

```diff
@@ -816,7 +818,7 @@
 				else if(MUT.name_prefix || MUT.name_suffix)
 					src.name = "\improper [MUT.name_prefix][growing.name][MUT.name_suffix] plant"
 			else
-				src.name = "\improper [growing.name] plant"
+				src.name = "\improper [growing.name] plant" //TODO: add optional suffix eg. "tree"
```

Comment:
Great TODO, we could definitely get rid of "Money Tree plant" and the like. Ideally create a GitHub Issue around this?

Reply:
Sounds good!

Reply:
Created issue.